src/index.css

```
@@ -746,6 +821,149 @@ body {
+  z-index: 1001;
+  display: flex;
+  align-items: center;
+  gap: 5px;
+  padding: 5px 12px;
+  border-radius: 20px;
+  border: none;
+  background: rgba(0, 0, 0, 0.04);
+  color: var(--text-muted);
+  font-size: 0.6rem;
+  font-weight: 700;
+  font-family: 'Outfit', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)